### DISCUSSION TITLE
Storage - GIT setup fails if the password has @

### DISCUSSION THREAD

**Discussion by aayushus** [requarks/wiki#3511] [Error / Bug Report]
**Describe the bug**
If the Github password has @ in it, it would mess up the Git setup.

**To Reproduce**
1. Change Github password, make sure to have @ in the password string.
2. Setup a new github storage, with basic authentication.
3. Setup and further syncs would fail.

**Expected behavior**
Sync should initialize.

**Screenshots**

**Host Info (please complete the following information):**
 - OS: [e.g. Ubuntu 20.04, Docker]
 - Wiki.js version: [e.g. 2.5.170]
 - Database engine: [e.g. postgres:11-alpine, docker]

**Comment by Smankusors** [comment]
Duplicate of #840

**Comment by lmmfranco** [comment]
Same problem occurs for most symbols. But `@` makes it obvious because it leaks the password on the error log.
Running `git remote -v` on the repo folder shows that the password was added without URL encoding.
